### ISSUE TITLE
Does SparseConv3d & SubMConv3d always produce the exactly same outputs?

### ISSUE THREAD

**Issue by zodiac000** [traveller59/spconv#584]
I found out by given the same set of inputs always receiving slightly different outputs. The diff is roughly about 10^-5. This is awkward. Is there any randomness involved in the implementation of 3DConv in spconv2.X or spconv1.X? If so, is there any way to avoid the randomness occuring.

**Comment by jisoo0-0** [comment]
same here. .. Did you solved the problem..??

**Comment by harrydobbs** [comment]
Same here - did either of you find a solution?

**Comment by x1y9** [comment]
same here. I have set torch.manual_seed and random.seed, still get changed outputs

**Comment by jisoo0-0** [comment]
Nah... I have not found the solution for this problem. So i have repeated experiment for multiple times.

**Comment by lysukhin** [comment]
+1. Seems like the source of the randomness is somewhere inside the implementation of the sparse convolutions
